packages/frontend/src/page/components/answer-form/index.tsx

```diff
@@ -127,7 +127,7 @@ export const AnswerForm = ({
         : question.bond.mul(2);
 
     const finalBond = useMemo(() => {
-        return !!bond && !!bond.value
+        return !!bond && !!bond.value && !isNaN(Number(bond.value))
```

Comment:
I'd use `parseInt` as it converts `null` and `undefined` to `NaN` instead of 0 (it logically makes a bit more sense)